src/types/index.d.ts

```diff
@@ -3,3 +3,43 @@ export global {
     gtag: any
   }
 }
+export interface SocialLinksProps {
```

Comment:
(also above): Everything that we called 'display props' or 'component props' moves down to `@/types/`. These types serve two purposes:

* They define what the component should implement.
* They define what the drupal->next-build mapping should output.

It would probably be sensible to break these up into multiple files.